src/components/ExpensesList.js

```diff
@@ -0,0 +1,27 @@
+import Expense from "./Expense";
+
+const ExpensesList = ({ expenses, removeExpense }) => {
+  const renderExpenses = () => {
+    const expenseItems = expenses.map(expense => (
+      <Expense
+        key={expense.id}
+        expense={expense}
```

Comment:
@MatthewBerglund: have you tried to spread props? e.g. use only `{...expense}`, instead of `expense={expense}`.

If you never tried it, check it out. It makes a lot of sense to me when I need to pass an object with many keys I need to use.

Reply:
I'm more familiar with spreading props within the consuming component itself. Is there an advantage to spreading props when you pass them instead of doing it the other way?

Reply:
the only advantage is that in the consuming components you can directly destructure your props `({ id, title, amount, date, removeExpense })` and use them without doing `expense.id`, `expense.title` and so on... 

Just wanted to be sure you know you can do it.